arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -322,33 +351,32 @@
       "properties": {
         "title": {
           "type": "string",
-          "description": "The title of the Azure ML web service."
+          "description": "The title of the web service."
         },
         "description": {
           "type": "string",
-          "description": "The description of the Azure ML web service."
+          "description": "The description of the web service."
         },
         "createdOn": {
           "type": "string",
-          "description": "The moment of time the Azure ML web service was created.",
+          "description": "Read Only: The date and time when the web service was created.",
```

Comment:
I mentioned this in the email thread as well, but I agree. I've filed a DocFx tool bug to the VSC team.